src/components/Filter/FilterInterfaces.tsx

```diff
@@ -9,6 +9,7 @@ export interface FilterListInterface {
   operation?: string | string[] | undefined;
   value?: any;
   path?: string;
+  logic?: string;
```

Comment:
and when using enum, you can use it here as a type

```suggestion
  logic?: Logic;
```

this will give you better understanding of what component expects, and will also add autocompletion in your editor